common/util.go

```diff
@@ -1,20 +1,18 @@
 package common
 
-import (
-	"net"
-	"net/url"
-)
-
 var AzcopyJobPlanFolder string
 var AzcopyCurrentJobLogger ILoggerResetable
 
 
 func VerifyIsURLResolvable(url_string string) (error) {
+	return nil
```

Comment:
Should we add a link to the issue or comment to why we are "removing" this check?